### PR TITLE
Better list-style-position example

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
I have made examples be longer, so it is shown how multi line text is aligned.
image
image

Examples come from [space.com](https://www.space.com/5853-nasa-memorable-missions.html)

### CODE REVIEW

File: live-examples/css-examples/lists/list-style-position.html [hunk 1]
```diff
@@ -21,11 +21,11 @@
         <div>
             <p>NASA Notable Missions</p>
             <ul id="example-element" class="transition-all">
-                <li>Apollo</li>
-                <li>Hubble</li>
-                <li>Chandra</li>
-                <li>Cassini-Huygens</li>
-                <li>Spitzer</li>
+                <li>Apollo 11: First Moon Men</li>
```
Comment (wbamberg) [inline live-examples/css-examples/lists/list-style-position.html:24]:
```suggestion
                <li>Apollo 11: First Human Landing</li>
```

File: live-examples/css-examples/lists/list-style-position.html [hunk 2]
```diff
@@ -21,11 +21,11 @@
...
+                <li>City in Space: The International Space Station</li>
+                <li>The Columbia Tragedy: Feb. 1, 2003</li>
+                <li>Great Observatory: The Hubble Space Telescope</li>
+                <li>Jan. 28, 1986: Challenger Accident</li>
```
Comment (wbamberg) [inline live-examples/css-examples/lists/list-style-position.html:28]: Maybe it's just me, but could we have fewer tragedies? Maybe:

```suggestion
                <li>Great Observatory: The Hubble Space Telescope</li>
                <li>Everlasting Mars Rovers</li>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Yes, this, is a useful change, thank you! I had a couple of wording suggestions.

**Comment by NiedziolkaMichal** [comment]
Thanks for review, those are good suggestions.

**Review by wbamberg — APPROVED** [review]
Thank you @NiedziolkaMichal !

**Pull request merged**